baseband/vdif/base.py

```diff
@@ -263,8 +268,9 @@ def _get_time(self, header):
 
     def __repr__(self):
         return ("<{s.__class__.__name__} name={s.name} offset={s.offset}\n"
-                "    nthread={s.nthread}, "
-                "samples_per_frame={s.samples_per_frame}, nchan={s.nchan},\n"
+                "    nthread={s._sample_shape.nthread}, "
```

Comment:
Still necessary because we want to print out the EDV and station.  I also added a `__repr__` method to `GSBStreamBase` (currently it prints the names of all raw data files in an ugly way...).